docs/content/intro/quickstart.md

```diff
@@ -150,6 +150,8 @@ $ kubectl --namespace brigade port-forward service/brigade-apiserver 8443:443 &>
 
 ### Log into Brigade
 
+> ⚠️ In this section, we'll be logging in as the root user. This option should typically be disabled in a production-grade Brigade deployment. Read more about user authentication [here](https://docs.brigade.sh/topics/administrators/authentication/).
```

Comment:
I don't think we should be using an absolute URL here. If the site is deployed to another domain (this happens, for instance, on the Netlify deployment previews) this will cause problems.

Reply:
Oh I didn't notice that earlier.